packages/server/src/automations/tests/scenarios/scenarios.spec.ts

```diff
@@ -157,4 +236,101 @@ describe("Automation Scenarios", () => {
     expect(results.steps[1].outputs.success).toBeTruthy()
     expect(results.steps[2].outputs.rows).toHaveLength(1)
   })
+
+  it("should query an external database for some data then insert than into an internal table", async () => {
+    let newTable = await config.createTable({
+      name: "table",
+      type: "table",
+      schema: {
+        name: {
+          name: "name",
+          type: FieldType.STRING,
+          constraints: {
+            type: FieldType.STRING,
+            presence: true,
+          },
+        },
+        age: {
+          name: "age",
+          type: FieldType.NUMBER,
+          constraints: {
+            type: FieldType.NUMBER,
+            presence: true,
+          },
+        },
+      },
+    })
+
+    let client: Knex
+
+    let db = await getDatasource(DatabaseName.MYSQL)
```

Comment:
Might be worth breaking this out into a function that can be re-used through the automation tests, the ability to create an external database with table, with rows, for querying purposes could be very useful if reusable.